app/frontend/src/styles/application/jobseekers/subjects-and-grades.scss

```diff
@@ -26,6 +26,10 @@
       .govuk-label {
         margin-bottom: govuk-spacing(3);
       }
+
+      .delete-button {
+        display: none;
```

Comment:
I don't understand this bit.

Reply:
my reasoning is that the delete button is never used or should appear on first row?